### DISCUSSION TITLE
Proper way to alter a copied Requester?

### DISCUSSION THREAD

**Discussion by steve-anthony-lee** [Opinionated-Blueprints/10-Books-Full-of-Rails#124] [General]
I've been making some city blocks and finally have a pattern I like, but what's the best way to copy my Iron Ore processing and paste it into a new block and switch it to Copper Ore processing? Should I just edit the name? Fiddle with some setting in the Train Station or one of the electronics?

**Comment by AspectLL** [comment]
Provider stations are dumb in the sense that they'll accept whatever material you're pumping into them with no changes in settings. For your requesters that are accepting iron ore, you can just paste a new requester station over the existing one and it'll give you a new gump to put in the settings you want. If it's a mixed requester station, you might need to look at the constant combinator that's attached to the stop and make sure that it's accepting all of the items you want accepted.

**Reply by steve-anthony-lee** [reply]
Doh! I never even thought of just dumping a small requester station on top! Thanks.
